<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchMainWindow</class>
 <widget class="QMainWindow" name="MatchMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>连连看</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/icon.png</normaloff>image/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="prbtime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {   border: 2px solid grey;   border-radius: 5px;   background-color: #FFFFFF;}QProgressBar::chunk {   background-color: #05B8CC;   width: 5px;}QProgressBar {   border: 2px solid grey;   border-radius: 5px;   text-align: center;}
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>提示:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeltip">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>重排:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelreset">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>得分:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tipButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(157, 220, 217);</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>80</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(157, 220, 217);</string>
    </property>
    <property name="text">
     <string>重排</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>排名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelscore">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>390</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>360</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>聊天室：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>710</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>710</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_G">
    <property name="title">
     <string>游戏G</string>
    </property>
    <addaction name="gameaction"/>
    <addaction name="restartaction"/>
    <addaction name="separator"/>
    <addaction name="exitaction"/>
    <addaction name="gameaction2"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>帮助A</string>
    </property>
    <addaction name="helpaction"/>
   </widget>
   <addaction name="menu_G"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="gameaction">
   <property name="icon">
    <iconset>
     <normaloff>image/24.png</normaloff>image/24.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="restartaction">
   <property name="icon">
    <iconset>
     <normaloff>image/exit.png</normaloff>image/exit.png</iconset>
   </property>
   <property name="text">
    <string>停止游戏</string>
   </property>
  </action>
  <action name="exitaction">
   <property name="icon">
    <iconset resource="image/im.qrc">
     <normaloff>:/000.jpg</normaloff>:/000.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="helpaction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="gameaction2">
   <property name="icon">
    <iconset>
     <normaloff>image3/17.png</normaloff>image3/17.png</iconset>
   </property>
   <property name="text">
    <string>开始2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image/im.qrc"/>
 </resources>
 <connections/>
</ui>
